access/views.py

```diff
@@ -260,6 +254,17 @@ def generated_exercise_file(request, course_key, exercise_key, exercise_instance
                     raise Http404()
     raise Http404()
 
+
+def _get_course_exercise_lang(course_key, exercise_key, lang_code):
+    (course, exercise) = config.exercise_entry(course_key, exercise_key, lang=lang_code)
+    if course is None or exercise is None:
+        raise Http404()
+    if not lang_code:
+        lang_code = course.get('lang', DEFAULT_LANG)
```

Comment:
Can we someway get this from the exercise? In other words, do we know the language a specific exercise is done with? In case where different language would be totally different exercise (for some unknown reason). Or does this actually even matter in case of single language for a exercise?